spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationInterpreter.scala

```diff
@@ -38,7 +39,7 @@ object StandardizationInterpreter{
     * @param df         Dataframe to be standardized
     * @param expSchema  The schema for the df to be standardized into
     */
-  def standardize(df: Dataset[Row], expSchema: StructType, inputType: String)
+  def standardize(df: Dataset[Row], expSchema: StructType, inputType: String, failFast: Boolean = false)
```

Comment:
`failFast` here means the same as `failOnInputNotPerSchema` in other places, right? Maybe let's have the same name here as well.